packages/ember-glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -2101,4 +2101,40 @@ moduleFor('Components test: curly components', class extends RenderingTest {
 
     this.assertText('initial value');
   }
+
+  ['@htmlbars a two way binding flows upstream when set during init (#11192)']() {
```

Comment:
Hmm. I think you are correct, but in this case the change events are not on the object that is running `init`, but on the thing that has passed in a two way bound property.  This was definitely one of the regressions introduced in 1.13 (same issue that caused `viewName` to not work properly).

FWIW - Doing this, does (properly) trigger a deprecation about causing a rerender.